test/Client.test.ts

```diff
@@ -176,6 +176,37 @@ describe('Client', () => {
         convos.forEach((m, i) => assert.equal(m.content, messages[i]))
       })
 
+      it('query historic messages', async () => {
+        const c1 = await testCase.newClient()
+        const c2 = await testCase.newClient()
```

Comment:
Is there a reason to not use the Alice and Bob Client instances that are automatically created for this test case?

 Initializing new clients will result in excess messages on the network as they publish contactBundles on initialization

Reply:
I'm not a huge fan of reusing state across tests tbh but I'll update this to be consistent with the others.

Reply:
My bad, I mistakenly thought that the clients were being generated once per test. Makes sense to me to start with a new client to ensure a clean state